### PR TITLE
General settings: add screenshots, make procedural

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

General settings: rephrase
- add screenshots
- rephrase steps about editing settings, add numbered steps
- Move YAML section into homeassistant integration docs
- add related topics

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/basic.markdown [hunk 1]
```diff
@@ -1,117 +1,33 @@
 ---
 title: "Setup basic information"
 description: "Setting up the basic info of Home Assistant."
+related:
+- docs: /integrations/homeassistant/
+- docs: /docs/configuration/
```
Comment (klaasnicolaas) [inline source/_docs/configuration/basic.markdown]:
```suggestion
  - docs: /integrations/homeassistant/
  - docs: /docs/configuration/
```

File: source/_integrations/homeassistant.markdown [hunk 2]
```diff
@@ -11,9 +11,130 @@ ha_domain: homeassistant
 ha_platforms:
   - scene
 ha_integration_type: system
+related:
+- docs: /docs/configuration/basic/
+  title: Basic information
+- docs: /docs/configuration/
```
Comment (klaasnicolaas) [inline source/_integrations/homeassistant.markdown]:
```suggestion
  - docs: /docs/configuration/basic/
    title: Basic information
  - docs: /docs/configuration/
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_docs/configuration/basic.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_docs/configuration/basic.markdown]: Thank you for the catch :fishing_pole_and_fish: :heart:

**Review by frenck — APPROVED** [review]
Nice, thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**